sdk/endpoints/online/managed/online-endpoints-managed-identity-sai.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Access Azure resources from an online endpoint with a system-assigned managed identity\n",
+    "## Define configuration for deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+     "name" : "define-configuration-1"
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import ( \n",
+    "    ManagedOnlineDeployment, ManagedOnlineEndpoint, \n",
+    "    Model, CodeConfiguration, Environment ) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "define-configuration-2"
+   },
+   "outputs": [],
+   "source": [
+    "sai_endpoint_name = \"my-sai-endpoint\"\n",
+    "sai_endpoint = ManagedOnlineEndpoint(\n",
+    "    name=sai_endpoint_name, \n",
+    "    auth_mode=\"key\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "define-configuration-3"
+   },
+   "outputs": [],
+   "source": [
+    "sai_deployment = ManagedOnlineDeployment(\n",
+    "    name=\"blue\", \n",
+    "    endpoint_name=sai_endpoint_name, \n",
+    "    model=Model(\n",
+    "        path=\"../../model-1/model/\"\n",
```

Comment:
Could you place these two notebooks into the managed-identities directory like `sdk/endpoints/online/managed/managed-identities` to ensure the relative path `../../model-1/` work properly?

Reply:
Change made